packages/client-sdk-nodejs/src/batchutils/batch-props.ts

```diff
@@ -0,0 +1,36 @@
+import {CacheGet, CacheSet, CacheDelete} from '@gomomento/sdk-core';
+import {CacheClient} from '../cache-client';
+
+export const defaultMaxConcurrentRequests = 5;
+
+export interface BatchGetRequest {
+  cacheClient: CacheClient;
+  cacheName: string;
+  keys: Array<string>;
+  maxConcurrentGets?: number;
+}
```

Comment:
as mentioned above I think you can get rid of all the request objects in this file. And you will want to create an Options type/interface for each one. For now all 3 options objects will probably all be the same so you can alias them, like what we do here:

https://github.com/momentohq/client-sdk-javascript/blob/d3b796575829f8406d42ae902cda04b379308487/packages/core/src/clients/IMomentoCache.ts#L62